Assets/Images/Tilemaps.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
-guid: ee09dc4536361794ca5c4553e4dcaedf
-folderAsset: yes
-DefaultImporter:
-  externalObjects: {}
-  userData: 
-  assetBundleName: 
-  assetBundleVariant: 
```

Comment:
@christerk  you keep adding this file back in, is there a Tilemaps file you have locally that hasnt been commited??